src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Attributes.cs

```diff
@@ -327,6 +328,8 @@ public static class GetCustomAttribute
     {
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/linker/issues/2078", typeof(PlatformDetection), nameof(PlatformDetection.IsTrimmedWithILLink))
```

Comment:
Wondered why this wasn't hit with naot and found out illinker has this as a hardcoded list. Weird, but okay I guess. Would be nice to move this to the same plan as all the other attributes at least. I'm not convinced anyone will want to work on 2078 anytime soon.